endpoints/openrtb2/amp_auction.go

```diff
@@ -510,6 +510,15 @@ func (deps *endpointDeps) parseAmpRequest(httpRequest *http.Request) (req *openr
 	// move to using the request wrapper
 	req = &openrtb_ext.RequestWrapper{BidRequest: reqNormal}
 
+	// upgrade to 2.6 here
```

Comment:
nitpick: recommend renaming to "normalize to openrtb 2.6" since the incoming request may already be 2.6.

nitpick: can be simplified to:
```
if err := openrtb_ext.ConvertUpTo26(req); err != nil {
  errs = append(errs, err)
} 
```